_includes/mini-curriculum/debugging/debugger.md

```diff
@@ -144,20 +144,17 @@ worry. We're about to learn better techniques.
 ## Pry and debugger
 
 We're going to use two gems. One is called Pry, which is a replacement
-for [irb][irb]. You'll have to `gem install pry`. It's not essential for
-debugging that you use Pry, but it will make life nicer.
-
-[irb]: http://en.wikipedia.org/wiki/Interactive_Ruby_Shell
-
-We're also going to use debugger (`gem install debugger`). This is a
-Ruby 1.9.3 gem, so make sure you have that version. You may also wish
+for [irb][irb]. You'll have to `gem install pry`. You may also wish
 to grab my [.pryrc][pryrc] file. Download
 this to your home (`~`) directory and rename it with an initial
-dot. They should be picked up and used by Pry and the Ruby debugger
-automatically.
+dot. It should be picked up and used by Pry automatically. It's not essential for
+debugging that you use Pry, but it will make life nicer.
 
+[irb]: http://en.wikipedia.org/wiki/Interactive_Ruby_Shell
 [pryrc]: https://github.com/appacademy/dotfiles/tree/master/dot
 
+We're also going to use debugger (`gem install debugger`). This is a
+Ruby 1.9.3 gem, so make sure you have that version.
```

Comment:
Oh golly gee, Ruby 1.9.3?! I didn't notice this. I don't think we have to worry about anyone using Ruby 1.8 at this point, so just remove this sentence. In fact, I'll just do it after I merge.